src/main/scala/vectorpipe/osm/package.scala

```diff
@@ -184,7 +185,12 @@ package object osm {
     *
     * @group conversion
     */
-  def features(ns: RDD[(Long, Node)], ws: RDD[(Long, Way)], rs: RDD[(Long, Relation)]): Features = {
-    PlanetHistory.features(ns, ws)
+  def features(
+    ns: RDD[(Long, Node)],
+    ws: RDD[(Long, Way)],
+    rs: RDD[(Long, Relation)],
+    storageLevel: Option[StorageLevel] = Some(StorageLevel.MEMORY_ONLY)
```

Comment:
Since it's a default arg already, it would probably result in less code overall if it read:
```scala
storageLevel: StorageLevel = StorageLevel.MEMORY_ONLY
```
Then you wouldn't have to deal with threading the `Option` through.

Unless there's a case where someone would specifically want to do *no* persisting? I can't see that being the typical case.

Reply:
Yeah, the reason I made it an `Option` was in case someone did want any persisting. I'm not sure how often that situation would arise, but I thought it would be good to have the option available.

Reply:
There are situations where persistence is not desired...if I have a very large dataset on a memory constrained cluster, certain caching strategies could cause the job to always fail with out of memory errors. In that case I would want to turn off persistence to try and make the job succeed at the sacrifice of having the job run slower.

Reply:
There is `StorageLevel.NONE` that can be used as default: https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/storage/StorageLevel.scala#L153 that would get rid of the option.

Reply:
Some history of why we're handling this at all: during the Tobler work, @echeipesh and I had to run the ingest umpteen times with very specific memory settings, partition settings, and persist/unpersist calls in many places to get the job to work. At that time, setting these persists in VectorPipe to `MEMORY_ONLY` was the only thing that ended up working, since `MEMORY_ONLY` persisted blocks are expected to be ejected automatically if later stages need the memory.